### DISCUSSION TITLE
Can a youtubeConfig prevents ratelimt?

### DISCUSSION THREAD

**Discussion by urfavteddybear** [lavalink-devs/Lavalink#992] [Q&A] [answered]
Hi there! I saw quite much issue that YouTube are rate limiting IP when using Lavalink, I do setup an IP rotation on my Lavalink but kinda curious, can a youtubeConfig prevent ratelimit too? or IP rotation is the only way to prevent this issue?

Thanks!

**Comment by devoxin** [comment, marked as answer]
As the example application.yml says:
>  [A youtubeConfig is] Required for avoiding all age restrictions by YouTube, some restricted videos still can be played without.

The only way to avoid ratelimiting is to use a large IPv6 block to distribute requests across.
